testsuite/shared/src/main/java/org/jboss/as/test/deployment/DeploymentInfoUtils.java

```diff
@@ -0,0 +1,797 @@
+         */
+        public String getCommand() {
+            return this.command;
+        }
+
+        /**
+         * Return joined string of parsed rows or output empty mark in case output is empty
+         *
+         * @return Return joined string of parsed rows or output empty mark in case output is empty
+         */
+        @Override
+        public String toString() {
+            if (this.isOutputEmpty()) {
+                return OUTPUT_EMPTY_MARK;
+            }
+            return String.join("\n", this.rows);
+        }
+
+        /**
+         * If is set selected server group
+         *
+         * @return If server group has set return true, else false
+         */
+        public boolean hasServerGroup() {
+            return this.serverGroup != null;
+        }
+
+        /**
+         * Get selected server group in list/info command
+         *
+         * @return Selected server group in list/info command
+         */
+        public String getServerGroup() {
+            return this.serverGroup;
+        }
+
+        /**
+         * Get information message about selected server group.
+         *
+         * @return Information message about selected server group
+         */
+        public String getServerGroupInfo() {
+            return this.serverGroup != null ? " for server group '" + this.serverGroup + "'" : "";
+        }
+
+        /**
+         * Get original command output.
+         *
+         * @return Raw command output.
+         */
+        public String getOriginalOutput() {
+            return this.originalOutput;
+        }
+
+        // &&&& BEGIN Method for additional information about processing verify check
+
+        /**
+         * Additional information about processing verify check.
+         *
+         * @param request Request of management trusted command to verify check.
+         */
+        public void setRequest(String request) {
+            this.request = request;
+            this.response = null;
+        }
+
+        /**
+         * Additional information about processing verify check.
+         *
+         * @return Request of management trusted command to verify check.
+         */
+        public String getRequest() {
+            return this.request;
+        }
+
+        /**
+         * Additional information about processing verify check.
+         *
+         * @param response Response of management trusted command to verify check.
+         */
+        public void setResponse(String response) {
+            this.response = response;
+        }
+
+        /**
+         * Additional information about processing verify check.
+         *
+         * @return Response of management trusted command to verify check.
+         */
+        public String getResponse() {
+            return this.response;
+        }
+    }
+
+    /**
+     * Container of Utils internal parameters, reduce of overloaded function of check
+     */
+    private static class InternalParameters {
```

Comment:
InternalParameters is a bit vague it seems that this is a parameter passed to check methods. "CheckParameters" is perhaps a more descriptive name?

Reply:
You have right.
Thank you. Done.